test/e2e-go/features/participation/onlineOfflineParticipation_test.go

```diff
@@ -247,11 +247,7 @@ func TestAccountGoesOnlineForShortPeriod(t *testing.T) {
 	t.Parallel()
 	a := require.New(fixtures.SynchronizedTest(t))
 
-	// Make the seed lookback shorter, otherwise will wait for 320 rounds
```

Comment:
I add `FasterConsenus` in this PR, and this test does not have a long wait in it.  I think the error was that it was cut and pasted from the test above (which has the same comment). That test _does_ speed up consensus, because it has a long wait.

So I think the comment was the mistake, so I deleted it.